src-core/src/spec/trajectory.rs

```diff
@@ -399,6 +399,9 @@ pub struct Trajectory {
     /// This includes 0, but the index of the last sample is never in this list even if the split toggle is set
     /// for the last waypoint
     pub splits: Vec<usize>,
+    /// The trackwidth of the robot,
+    /// used in robot code for differential drive calculations
+    pub trackwidth: f64,
```

Comment:
Either this should be optional or we have to define it for swerve

Reply:
Yeah, requiring it for swerve isnt bad, the data is already in robot config

Reply:
Is it? Which distance between modules is used?

Reply:
between front left and front right ig, we can document that behavior